.github/ISSUE_TEMPLATE.md

```diff
@@ -1,12 +1,17 @@
-Make sure you are running the latest version of Home Assistant before reporting an issue.
-
-You should only file an issue if you found a bug. Feature and enhancement requests should go in [the Feature Requests section](https://community.home-assistant.io/c/feature-requests) of our community forum:
-
+<!-- READ THIS FIRST:
+- Make sure you are running the latest version of Home Assistant before reporting an issue
+- Do not report issues for components if you are using custom components
+- Frontend (UI) related issues have to be reported here: https://github.com/home-assistant/home-assistant-polymer/issues
```

Comment:
Not sure, but I think most people that report issues won't know whether an issue is core/frontend-related. So maybe there will suddenly be a bunch of core issues created in the frontend repo...